src/test/cypress/e2e/integration/app-frontend/redirect.js

```diff
@@ -18,7 +18,7 @@ describe('Redirect', () => {
     cy.get(appFrontend.altinnError).should('contain.text', texts.missingRights);
   });
 
-  it.only('Unknown error page', () => {
+  it('Unknown error page', () => {
```

Comment:
I think the titles of the tests should have a description of what they do.
"it unknown error page" isn't really a sentence.
Is it supposed to be "It renders the 'unknown error page'"?

Reply:
agree. the name of the test is renamed now.